style/ruby/README.md

```diff
@@ -3,41 +3,23 @@ Ruby
 
 [Sample](sample.rb)
 
-* Avoid conditional modifiers (lines that end with conditionals).
```

Comment:
El benefit que yo veo es que hace que el code sea más fácil de leer. Que todos los if statements estén estructurados de la misma forma, you get in the habit.

Me apunto a hacerlo como vos querrás.